tests/integration_test_dag.py

```diff
@@ -0,0 +1,210 @@
+import pathlib
```

Comment:
Did you consider also adding `truncate`. `render` and `stats_check` the `integration_test_dag.py`?

Reply:
@tatiana I want to add those in future PRs. I want to start with a working integration test DAG and then as we fix the test cases we will add all of those features in.